### PR TITLE
add repo documentation

### PR DESCRIPTION
**Pull request by sonasingh46**
This PR adds the following documentation:
1. Adds the repo overview readme.
2. Adds developer guide.
3. Adds a quick start guide for pool provisioning.
4. Adds CI badges.
5. Add pool operation tutorial for stripe and mirror pool config.

Signed-off-by: Ashutosh Kumar <ashutosh.kumar@mayadata.io>

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
```
Comment (kmova) [inline CODE_OF_CONDUCT.md:1]: This should be simple like this: https://github.com/openebs/libcstor/blob/master/CODE_OF_CONDUCT.md

(The repo's that don't follow this above simple format, needs to be updated).

File: README.md [hunk 2]
```diff
@@ -1,2 +1,95 @@
...
+We consider issues also as a part of contribution to the project.
+If you want to raise any issue for cstor-operators please do that at [openebs/openebs].
+We are tracking issues for all the OpenEBS components at the same place.
+If you are unsure about how to proceed, do not hesitate in communicating in 
+OpenEBS community slack [channel]. 
+
+See [contributing](#contributing) section to learn more about how to contribute to cstor-operators.
+
+
+## Contributing
```
Comment (kmova) [inline README.md:71]: Push this into CONTRIBUTING.MD file

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (sonasingh46) [reply CODE_OF_CONDUCT.md:1]: Thanks. Will update that.

File: README.md (hunk 2 above)
Comment (sonasingh46) [reply README.md:71]: This is just an intro section... Rest all are pushed to the developer guide.

**Pull request merged**